### PR TITLE
curating sample pipelines to expand on

### PR DESCRIPTION
**Pull request by animeshsingh**
adding some pipelines snippet code to show kfserving usage. Will be expanded with more robus examples moving forward [added some missing license headers which caught the eye]

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/340)

### CODE REVIEW

File: docs/samples/pipelines/sample-tf-pipeline.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+@dsl.pipeline(
+  name='kfserving pipeline',
+  description='A pipeline for kfserving.'
+)
+def kfservingPipeline(
+    action = 'create',
+    model_name='tensorflow-sample',
+    default_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
+    canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
```
Comment (yuzisun) [inline docs/samples/pipelines/sample-tf-pipeline.py]: the uri should be different for canary

File: docs/samples/pipelines/sample-custom-model.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp.dsl as dsl
+from kfp import components
+import json
+
+
+kfserving_op = components.load_component_from_file('component.yaml')
```
Comment (yuzisun) [inline docs/samples/pipelines/sample-custom-model.py]: what's `component.yaml` ?

File: docs/samples/pipelines/sample-tf-pipeline.py (hunk 1 above)
Comment (animeshsingh) [reply docs/samples/pipelines/sample-tf-pipeline.py]: updated

File: docs/samples/pipelines/sample-custom-model.py (hunk 2 above)
Comment (animeshsingh) [reply docs/samples/pipelines/sample-custom-model.py]: defines what will serve the request in pipeline. Added a todo for updating with raw github URL from KFP repo once the component is updated there

**Comment by jinchihe** [comment]
@animeshsingh Do we plan to add the pipeline sample in the kubeflow Pipeline repo?

**Comment by animeshsingh** [comment]
Not these - there shall be one final good sample there, but the ones here will be representative of multiple scenarios we create.

**Pull request merged**